js/components/SetAReminder/SetAReminder.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  Text,
+  View,
+  Modal,
+  TouchableOpacity,
+  Switch,
+  StyleSheet,
+  DatePickerIOS
+} from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import styles from "./styles";
+
+class SetAReminder extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chosenDate: new Date(),
+      chosenTime: "8:30",
+      modalVisible: false,
+      reminderSwitch: false
+    };
+  }
+  setDate = newDate => {
+    console.log(newDate)
+    this.setState({ chosenDate: newDate });
+  };
+  toggleReminderSwitch = value => {
+    this.setState({ reminderSwitch: value });
+  };
+  setModalVisible(visible) {
+    this.setState({ modalVisible: visible });
+  }
+
+  render() {
+    return (
+      <View>
+        <Modal
+          animationType="fade"
+          transparent={true}
+          visible={this.state.modalVisible}
+        >
+          <View style={styles.modalContentContainer}>
+            <LinearGradient
+              useAngle={true}
+              angle={180}
+              angleCenter={{ x: 0.5, y: 0.5 }}
+              colors={["#F3E3D8", "#fff"]}
```

Comment:
Linear gradient colours should be abstracted to global styles. Talk with @ajaguasin to get this sorted.